python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt/sling/asset_decorator.py

```diff
@@ -88,7 +88,11 @@ def my_assets(context, sling: SlingResource):
                 key=dagster_sling_translator.get_asset_key(stream),
                 deps=dagster_sling_translator.get_deps_asset_key(stream),
                 description=dagster_sling_translator.get_description(stream),
-                metadata=dagster_sling_translator.get_metadata(stream),
+                metadata={  # type: ignore
```

Comment:
Surprised that a type hint here is needed here

Reply:
This is the error:
> Argument of type "dict[str, Any | DagsterSlingTranslator | Mapping[str, Any]]" cannot be assigned to parameter "metadata" of type "RawMetadataMapping | None" in function "__new__"    Type "dict[str, Any | DagsterSlingTranslator]
<img width="1821" alt="image" src="https://github.com/dagster-io/dagster/assets/5807118/10879105-02a1-4c14-a94c-ddced6f9b2c0">